ui-v2/app/templates/dc/services/-upstreams.hbs

```diff
@@ -4,19 +4,23 @@
     items=item.Proxy.Upstreams as |item index|
 }}
     {{#block-slot 'header'}}
-      <th>Destination Name</th>
-      <th>Destination Type</th>
-      <th>Local Bind Port</th>
+      <th>Upstream</th>
+      <th>Datacenter</th>
+      <th>Type</th>
+      <th>Local Bind Address</th>
     {{/block-slot}}
     {{#block-slot 'row'}}
-      <td data-test-destination-name>
-        <a>{{item.DestinationName}}</a>
+      <td>
+        <a data-test-destination-name>{{item.DestinationName}}</a>
+      </td>
+      <td data-test-destination-datacenter>
+        {{item.Datacenter}}
       </td>
       <td data-test-destination-type>
           {{item.DestinationType}}
       </td>
-      <td data-test-local-bind-port>
-          {{item.LocalBindPort}}
+      <td data-test-local-bind-address>
+        {{item.LocalBindAddress}}:{{item.LocalBindPort}}
```

Comment:
Could it be confusing we call this an address where it is really a combination of address and port? I guess we do that all over the place but worth saying out loud.

Reply:
Yeah maybe, can defo see your point. I'd love to think of another word to describe ip-address+port, 'address' is the best word I can think of. Generally if you want to know where something is i.e. its address, you need 2 things the ip address and the port, kind of like street address plus house number.

Maybe thinking of it this way:

> where it is really a combination of address and port?

where it is really a combination of _ip_ address and port. _ip_ address + port = address ?